src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
```

Comment:
It seems that you are trying to link a SCSS file directly in your HTML. Browsers do not support SCSS files natively, you need to compile your SCSS to CSS and then link the resulting CSS file in your HTML.